docker/README.md

```diff
@@ -47,7 +47,7 @@ The DB will initialize itself upon startup via the init container (superset-init
 To run the container, simply run:
 
 ```bash
-docker-compose up
+docker-compose --env-file docker/.env up
```

Comment:
Let's not require this. Being able to just run "up" is part of what makes compose so easy. I don't think we need to go as far as this, as other DB types will likely require more options to be configured in the yaml. If you need multiple DBS, simply create another compose file and in include it with ur 'up'.